sklearn/gaussian_process/tests/test_gpr.py

```diff
@@ -23,7 +23,7 @@
 from sklearn.utils._testing \
     import (assert_array_less,
             assert_almost_equal, assert_array_almost_equal,
-            assert_array_equal, assert_allclose)
+            assert_allclose)
```

Comment:
Since imports get modified, can you please modify them as follows?
```python
from sklearn.utils._testing import (
    assert_array_less,
    assert_almost_equal,
    assert_array_almost_equal,
    assert_allclose,
)